README.md

```diff
@@ -1,10 +1,155 @@
 # Community
 
-## Become a member of rust-vmm
+## What is rust-vmm
+
+rust-vmm is an open-source project that enables the community to build custom
+Virtual Machine Monitors (VMMs) and Hypervisors. It offers a base of
+virtualization packages that other projects can use for rapidly developing
```

Comment:
s/other projects/any project/
s/use for rapidly developing/benefit from in order to quickly develop/